relaybot.py

```diff
@@ -52,7 +53,15 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        optionAsBoolean = { "": False, "false": False, "no": False, "true": True, "yes": True }

```

Comment:
Empty string as `False` seems undesirable to me but other than that this is fine.